libnmstate/validator.py

```diff
@@ -400,6 +436,12 @@ def _validate_linux_bond(original_desired_state, current_state):
             _assert_bond_config_is_legal(iface_state, original_iface_state)
 
 
+def _validate_vlan(original_desired_state, current_state):
+    _assert_vlan_mtu_not_greater_than_base_iface(
+        original_desired_state, current_state
+    )
+
+
```

Comment:
In this case the function only contains another function. There is no need for this alias, IMO.